extensions/wikia/TriviaQuizzes/TriviaQuizzesHooks.class.php

```diff
@@ -1,13 +1,24 @@
 <?php
 
-class TriviaQuizzesHooks extends WikiaController {
+class TriviaQuizzesHooks {
 	public static function onBeforePageDisplay( OutputPage $out ) {
+		wfProfileIn( __METHOD__ );
 		global $wgEnableTriviaQuizzesExt, $wgTriviaQuizzesEnabledPages;
 
 		if ( $wgEnableTriviaQuizzesExt && in_array( $out->getTitle()->getPrefixedText(), $wgTriviaQuizzesEnabledPages ) ) {
 			$out->addModules( 'ext.wikia.TriviaQuizzes' );
 		}
 
+		wfProfileOut( __METHOD__ );
+		return true;
+	}
+
+	public static function onWikiaSkinTopScripts( Array &$vars, &$scripts ) {
+		wfProfileIn( __METHOD__ );
+		$td = (new ThemeSettings())->getSettings();
+		$vars['wgTriviaQuizzesPrimaryColor'] = $td['color-community-header'];
```

Comment:
This is probably fine, but I'm wondering if there's another naming syntax for these kinds of variables that aren't really wikia global (`$wgVariable`) types but are sort of faked for the front-end. It probably doesn't matter though.